rllib/offline/utils.py

```diff
@@ -0,0 +1,60 @@
+import tree
```

Comment:
Let's move these to `rllib/utils/__init__.py`. Analogous to e.g. `force_list`. These are generic utils, not specific to offline RL.

Reply:
Will do. Didn't see another usage of these, but maybe could be used for transporting `MA-batches` to `Learner`s

Reply:
Crazy, I really just found one in one of my PRs :D
It's a very useful function!

E.g. you have a multi-agent obs space with all the agent keys. However, some obs published by the env do not contain always all agent IDs, so now we have to loop through the available agent IDs in the obs and match them with the individual spaces (e.g. complex dict space per agent) under the available agent IDs.